### ISSUE TITLE
CAPTCHA on password fail

### ISSUE THREAD

**Issue by plehegar** [w3c/webex#5]
If people attempt an incorrect password, it presents a CAPTCHA on subsequent attempts. A simple and solveable password problem can become an absolute barrier because of this, and people miss calls because even after obtaining the correct password (e.g., in IRC) they are unable to pass the CAPTCHA to join the call.

This is a failure of conformance to WCAG 2.0 and therefore of W3C site policy. Unfortunately it requires a fix from Cisco which has declined to address the issue. Short of a fix from Cisco or a switch to a better telephony solution, the most optimal fix is to avoid the CAPTCHA by better password support as described above.
